src/stream.jl

```diff
@@ -41,11 +41,15 @@ function create_group(
     config::AWSConfig,
     # this probably won't collide, most callers should add identifying information though
     log_group_name::AbstractString="julia-$(uuid4())";
-    tags::AbstractDict{<:AbstractString, <:AbstractString}=Dict{String, String}()
+    tags::Union{AbstractDict{<:AbstractString, <:AbstractString}, Nothing}=nothing,
 )
-    tags = Dict{String, String}(tags)
+    if tags === nothing
```

Comment:
Why not just do `isempty(tags)`?

Reply:
Also, if an empty dict causes the failure don't you still need to do the check I mentioned above?

Reply:
Good point, I'll do that instead